common/src/main.c

```diff
@@ -55,6 +57,9 @@ static struct Menu MENU = {
 #ifdef SPIFLASH_BASE
         MENU_ITEM('8', "SPI Flash Debugging", spiflash_menu),
 #endif
+#ifdef CONFIG_CPU_VARIANT_FOMU
```

Comment:
@JosephBushagour  here too, I think it makes sense to use the other config flag.

Reply:
Good catch! Fix pushed :).